### DISCUSSION TITLE
Improve 'Running ElastAlert for the first time'

### DISCUSSION THREAD

**Discussion by ferozsalam** [jertel/elastalert2#199] [Ideas]
The documentation site contains a page on '[Running ElastAlert for the first time](https://elastalert2.readthedocs.io/en/latest/running_elastalert.html#)'. While there's nothing obviously incorrect with the page, judging from the questions being asked in this Discussion forum, most people are using the Dockerised version of the software. I think it would be useful for the page to focus on running ElastAlert in Docker, perhaps taking the instructions from the README as a starting point and improving from there.

In addition, it might be useful to mention k8s configuration and the Helm chart, as we seem to get frequent questions about that as well. 

As usual, I would like to get any opinions or suggestions from @jertel and @nsano-rururu before I start any work.

**Comment by nsano-rururu** [comment]
I think you need to use the following three configurations:

- Running elastalert2 on k8s configuration and Helm chart
- Running elastalert2 on docker
- Running elastalert2 on direct installation

If you do not leave a description of the direct installation, there is a concern that the questions of those who want to use the direct installation will increase.
I am also the maintainer of [praeco](https://github.com/johnsusek/praeco) and [elastalert-server](https://github.com/johnsusek/elastalert-server).
I wrote in the readme that praeco and elastalert-server should be used in docker to recommend them in docker, but I'm inevitably asked if I didn't say that it works with direct installation. Since it took time to deal with it, the method of installing directly is described in detail at the bottom.
I think you should leave the description of direct installation.

**Reply by jertel** [reply]
I agree with both posts. Keep what we have now and move it to the third runtime method, then add the two new ones first and second, since most people use them. Great idea, thanks for offering to work on that!

**Comment by ferozsalam** [comment]
Hey @jertel would you be open to making me a collaborator on this repo? There are a few things I would like to do (categorising raised issues, setting up a stalebot and a dependency update bot, adding automated linting to pull requests) that I can't do with my current permissions. It's mostly organisational stuff that I think would make life easier for contributors/maintain a higher standard of the codebase.

I promise not to take any significant unilateral actions without discussion beforehand :) and am also happy for you to remove me at any point if you feel like I'm not really contributing much or not improving things. Obviously no issue if you don't think it's a good idea!

**Reply by jertel** [reply]
Yes. Thanks for all your refactoring work and assistance on the discussion forum!
